warehouse/query-core/src/main/java/datawave/query/index/lookup/RangeStream.java

```diff
@@ -457,12 +456,7 @@ public ScannerStream visit(ASTEQNode node, Object data) {
                 
                 if (setCondenseUids) {
```

Comment:
Is setCondenseUids synonymous with parseRootUids?

Reply:
No, not exactly...